client/eip712.go

```diff
@@ -0,0 +1,15 @@
+package client
+
+import (
+	"github.com/evmos/ethermint/client/eip712"
+	"github.com/spf13/cobra"
+)
+
+func EIP712Commands() *cobra.Command {
```

Comment:
nit, comment

Reply:
done

Reply:
@yihuang 🙏 Feel free to resolve conversations if the solution is trivial